ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/ConfigRenderer.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.runner.common
+
+import com.typesafe.config.{ConfigRenderOptions, ConfigValue}
+
+object ConfigRenderer {
+  val renderOptions: ConfigRenderOptions = ConfigRenderOptions
+    .defaults()
+    .setOriginComments(false)
+    .setComments(false)
+    .setJson(false)
+    .setFormatted(true)
+
+  private def toConfig(
+      config: Config
+  )(implicit writer: pureconfig.ConfigWriter[Config]): ConfigValue =
+    writer.to(config)
+
+  def render(configValue: ConfigValue): String =
+    configValue.render(renderOptions)
+
+  def render(config: Config): String = {
```

Comment:
Sometimes care needs to be taken to ensure passwords and the like are not written to log files.

Reply:
Yes, this would need to be part of `PureConfigReaderWriter` so writer hides secrets. 

Currently this class is not actually used, I think I will drop it from this PR, as it was required only to generate initial default config file for the conformance tests.